docs/tutoriales/publicacion-de-paquetes.md

```diff
@@ -0,0 +1,87 @@
+---
+id: publicacion-de-paquetes
+title: Publicación de Paquetes
+sidebar_label: Publicación de Paquetes
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Publicación de Paquetes
+La modularización en el contexto del software es importante para la separación de código y archivos que tengan relevancia entre sí, así como también facilitar el consumo de módulos que son comunes entre 2 o más proyectos. Aunque existen distintos tipos de paquetes y formas de publicarlos, este tutorial se basa en la publicación de paquetes de NPM.
+
+## Generación del Personal Access Token
+Para poder publicar paquetes, es necesario generar un *Personal Access Token* o **Token de Acceso Personal**. Este token puede ser generado al seguir los siguientes pasos:
+
+1. Diríjase a la opción de **Ajustes** o *Settings*
+    <p style={{ align: "center" }}>
+      <img src={ useBaseUrl( '/img/publicacion-de-paquetes/GoToSettings.png' )} width="100%" />
+     </p>
+1. Una vez se se encuentra en el panel de configuraciones, presione el botón *Developer Settings* o **Configuraciones de Desarrollador**
+    <p style={{ align: "center" }}>
+      <img src={ useBaseUrl( '/img/publicacion-de-paquetes/DeveloperSettings.png' )} width="100%" />
+    </p>
+1. Ahora, presione la opción **Tokens de Acceso Personal** o *Personal Access Tokens*
+    <p style={{ align: "center" }}>
+      <img src={ useBaseUrl( '/img/publicacion-de-paquetes/PersonalAccessToken.png' )} width="100%" />
+    </p>
+1. Una vez presionada la anterior opción, presione el botón **Generar nuevo token** o *Generate new token*
+    <p style={{ align: "center" }}>
+      <img src={ useBaseUrl( '/img/publicacion-de-paquetes/GenerateNewToken.png' )} width="100%" />
+    </p>
+1. Ahora seleccioone el *scope* o **ámbito** adecuado para el token, como se muestra en la siguiente imagen
+    <p style={{ align: "center" }}>
+      <img src={ useBaseUrl( '/img/publicacion-de-paquetes/TokenScope.png' )} width="100%" />
+    </p>
+1. Finalmente, presione el botón mostrado en la siguiente imagen (ubicado al final de la página) para obtener el token
+    <p style={{ align: "center" }}>
+      <img src={ useBaseUrl( '/img/publicacion-de-paquetes/GenerateTokenButton.png' )} width="100%" />
+    </p>
+**El token se muestra una vez este es generado; proceda a copiarlo en un lugar seguro, ya que no se vuelve a mostrar.**
+
+## Registro de Paquetes de Github
+*Github Package Registry* es un registro de paquetes que entre sus ventajas ofrece:
+- Manejo de las mismas credenciales que se utilizan en Github
+- Posibilidad de integración con *Github Actions* y *Webhooks*
+- Paquetes privados que se mantienen dentro del mismo ámbito o *scope* de la organización
```

Comment:
During our pre-practica process we took some decisions, one of them were to add a final point to each sentence in a list, not sure if it will still be persistent

Reply:
Fixed!